amplipi/streams.py

```diff
@@ -1124,9 +1124,10 @@ def info(self) -> models.SourceInfo:
 
 class LMS(PersistentStream):
   """ An LMS Stream using squeezelite"""
-  def __init__(self, name: str, server: Optional[str] = None, disabled: bool = False, mock: bool = False):
+  def __init__(self, name: str, server: Optional[str] = None, disabled: bool = False, mock: bool = False, deallocatable: bool = True):
```

Comment:
I was under the impression `deallocateable` would keep streams from being deleted.  At this point it needs to be a property of a base stream.

Reply:
I'm debating this a little now in my head though, because the intention is to keep the LMS stream running in the background.

Reply:
I debated with this a bit too and would love to riff on this a bit IRL; is this a property we'd want for streams other than these LMS clients, to create some sort of persistence all the time on other long-running streams?

Reply:
We might want this for other streams, although I don't have a concrete use case in mind.

Tomorrow work to riff on this?

Reply:
You can also just call me now